exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/xsort/TestSimpleExternalSort.java

```diff
@@ -147,7 +152,7 @@ public void sortOneKeyDescendingExternalSortLegacy() throws Throwable {
   }
 
   private void sortOneKeyDescendingExternalSort(boolean testLegacy) throws Throwable {
-    FixtureBuilder builder = ClusterFixture.builder()
+    FixtureBuilder builder = ClusterFixture.builder(dirTestWatcher)
```

Comment:
Is there a case where the tmp directories are optional? The Drillbit will always have to be configured with a <b>drill.tmp-dir</b> at minimum, so I think it should be required to use a BaseDirTestWatcher with a ClusterFixture. I could move passing the dirTestWatcher to the build method instead of the constructor. Let me know what you think.